bigquery_etl/query_scheduling/generate_airflow_dags.py

```diff
@@ -95,16 +95,41 @@ def get_dags(project_id, dags_config):
                 else:
                     if CHECKS_FILE in files:
                         checks_file = os.path.join(root, CHECKS_FILE)
-                        checks_task = copy.deepcopy(
-                            Task.of_dq_check(checks_file, dag_collection=dag_collection)
-                        )
-                        tasks.append(checks_task)
-                        task_ref = TaskRef(
-                            dag_name=task.dag_name,
-                            task_id=task.task_name,
-                        )
-                        checks_task.upstream_dependencies.append(task_ref)
+                        with open(checks_file, "r") as file:
+                            file_contents = file.read()
+                        # check if file contains fail and warn and create checks task accordingly
+                        if "#fail" in file_contents:
```

Comment:
Would it be useful to add some sort of delimiter to the checks list in `checks.sql` so that we could do a split and then bucket error checks separately from "warning" (soft fail) checks. so that we could run them as two separate tasks? This would allow us to run "warning" checks prior to running error checks?

Reply:
I guess currently we assume that if no marker is not specified for a test then `#fail` should be the default. So there could be files that have no `#fail` explicitly in them, but still a bunch of checks that should fail.